multi-node/generic/worker-install.sh

```diff
@@ -150,6 +209,8 @@ kind: Pod
 metadata:
   name: kube-proxy
   namespace: kube-system
+  annotations:
```

Comment:
# TODO, once `privileged` works sufficiently well in rkt, switch stage1 back to `stage1-coreos`